lib/src/html_generator.dart

```diff
@@ -466,7 +466,7 @@ abstract class TemplateData {
   String _layoutTitle(String name, String kind, bool isDeprecated) {
     if (kind.isEmpty) kind =
         '&nbsp;'; // Ugly. fixes https://github.com/dart-lang/dartdoc/issues/695
-    String str = '<div class="kind">$kind</div>';
+    String str = '<span class="kind">$kind</span>';
```

Comment:
Not part of your changes, but can you add braces around the if statement body above? Or move the comment one line above. It's formatting oddly.